Gems/Multiplayer/Code/Include/Multiplayer/NetworkTime/INetworkTime.h

```diff
@@ -60,6 +64,10 @@ namespace Multiplayer
         //! @param rewindConnectionId the rewinding ConnectionId 
         virtual void AlterTime(HostFrameId frameId, AZ::TimeMs timeMs, AzNetworking::ConnectionId rewindConnectionId) = 0;
 
+        //! Alters the current Host blend factor. Used to drive interpolation in rewound states.
+		//! @param blendFactor the blend factor to use
```

Comment:
Indentation looks a bit wonky.. also why not add the blend factor as a parameter to AlterTime itself?

Reply:
That's what I get for adding comments in my diff tool. I did this separately to make AlterTime a more intentional action rather than a part of AlterTime. For example, when we're adjusting the Client's time based on entity update receipt, we don't want a blend factor. That case was the biggest concern in my mind.